packages/dcos-history/extra/history/server.py

```diff
@@ -90,4 +92,10 @@ def start():
 
     state_buffer = BufferCollection(os.environ['HISTORY_BUFFER_DIR'])
     BufferUpdater(state_buffer, headers_cb).run()
-    app.run(host='0.0.0.0', port=int(os.environ.get('PORT', '15055')))
+
+
+def start():
```

Comment:
Nit: changing to gunicorn instead of /opt/mesosphere/bin/gunicorn doesn't help the fact that dcos_history_conf.py is not necessarily in working dir, but that might just be splitting hairs